### PR TITLE
fix: Add location to `ics` file

### PR DESCRIPTION
**Pull request by joeauyeung**
## What does this PR do?

This PR adds the `LOCATION` field to the ics file Cal.com generates.

Fixes # (issue)

## Requirement/Documentation

- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- Create a booking with a conferencing app. Check the ics file that was sent with the booking confirmation email. The `LOCATION` field should be included
- Do this for phone numbers, address, and a custom link
## Mandatory Tasks

- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

File: packages/lib/test/builder.ts [hunk 1]
```diff
@@ -176,7 +179,7 @@ export const buildCalendarEvent = (event?: Partial<CalendarEvent>): CalendarEven
     customInputs: {},
     additionalNotes: faker.lorem.paragraph(),
     organizer: buildPerson(),
-    videoCallData: buildVideoCallData(),
+    ...(!omitVideoCallData && { videoCallData: buildVideoCallData() }),
```
Comment (zomars) [inline packages/lib/test/builder.ts:182]: Just curious. Why is this needed?

Comment (joeauyeung) [reply packages/lib/test/builder.ts:182]: When generating the ics string we look for video call data first and then the location. Since video call data was always included in the event builder the ics string would always fallback to the video call url.

**Review by PeerRich — DISMISSED** [review]

File: packages/emails/lib/generateIcsString.ts [hunk 2]
```diff
@@ -93,6 +96,7 @@ const generateIcsString = ({
           }))
         : []),
     ],
+    location,
```
Comment (Udit-takkar) [inline packages/emails/lib/generateIcsString.ts:99]: Type checks are failing

**Review by Udit-takkar — APPROVED** [review]
